apps/valkyrie/config/integration.exs

```diff
@@ -51,3 +51,6 @@ config :valkyrie, :brook,
       namespace: "valkyrie:view"
     ]
   ]
+
+config :profiling,
```

Comment:
Elixir convention usually uses this like `config :app_or_library, key: value`. Since profiling isn't a library, we could add this to the configuration of valkyrie.